webapp/src/containers/SearchContainer.js

```diff
@@ -15,6 +15,16 @@ class SearchContainer extends React.Component {
     projects: PropTypes.array
   }
 
+  constructor(...args) {
+    super(...args)
+
+    const { minX, minY, maxX, maxY } = parcelUtils.getBounds()
+    this.coordinates = parcelUtils
+      .generateMatrix(minX, minY, maxX, maxY)
+      .map(coordinate => ({ name: coordinate }))
+      .sort((a, b) => a.name.length - b.name.length)
```

Comment:
Is this a perf improvement?
If so, I think that loosing a properly named method is not really necessary, we could just do:

`this.coordinates - this.getCoordinates()`

to achieve the same end result.